### DISCUSSION TITLE
Visual Studio 2022 Backspace Issue

### DISCUSSION THREAD

**Discussion by rngkGit** [community/community#13925] [Copilot Conversations]
Every time I run my Visual Studio 2022 and begin typing, the application runs just fine. But, when I immediately hit backspace, the application will freeze and crash. After I launch the application again, it says that Copilot has encountered a crash, and recommends for it to be disabled. Anyone else having this issue? My file is C# and I have the unity game engine library.
![image](https://user-images.githubusercontent.com/64713406/160919547-1f760485-7a98-42a7-a977-c2f478d4c0db.png)

**Comment by jeremi-przeorek** [comment]
I have the same exact issue

**Comment by streepje8** [comment]
I have this issue, it also happens without unity in a brand new console application project. It also seems to happen when trying to type anything after a while

**Comment by CharlotteAltima** [comment]
I have this problem with all languages

**Comment by thekim1** [comment]
Same for me but it is both the space and backspace keys

**Comment by booltechnologies** [comment]
Same here.

**Comment by kzu** [comment]
Fixed in latest release (v1.10.0.3 or later). Please update and give it a try!
